modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,104 @@
+It provides connection pool status information as well as connection handling, which indicates when the pool retrieves and returns connections.
+|===
+
+== Verbose Logging
+
+Use verbose logging to troubleshoot SDK components to get information regarding their status or behavior.
+Because of the high impact to application performance, limit verbose logging to testing and troubleshooting operations.
+
+Enable verbose logging in Anypoint Studio or Anypoint Runtime Manager if using CloudHub APIs.
+In both cases, the application's log file logs the information.
+
+=== Enable Verbose Logging in Anypoint Studio
+
+The following steps describe how to enable verbose logging using Anypoint Studio:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+
++
+image:enable-sdk-logging-studio.png[enable-sdk-logging-studio]
++
+
+. Add an `<AsyncLogger>` tag inside the `<Loggers>` tag:
+
++
+[source,xml,linenums]
+----
+<Loggers>
+    ...
+    <AsyncLogger name="<component's fully qualified name>" level="<logging level>"/>
+    ...
+</Loggers>
+----
++
+
+Use the following table as reference to specify the component's fully qualified name:
+
+[%header,cols="2,2,1"]
+|===
+| Name | Fully Qualified Name | Log Level
+| Source|`org.mule.runtime.module.extension.internal.runtime.source.ExtensionMessageSource`| `DEBUG`
+| Polling Source|`org.mule.runtime.module.extension.internal.runtime.source.poll.PollingSourceWrapper`| `DEBUG`
+| Polling Source Watermarking|`org.mule.runtime.module.extension.internal.runtime.source.poll.PollingSourceWrapper`| `TRACE`
+| Pooling Connection Provider|`org.mule.runtime.core.internal.connection.PoolingConnectionHandler`
+`org.mule.runtime.core.internal.connection.PoolingConnectionManagementStrategy`| `DEBUG`
+|===
+
+The following example shows how to add a fully qualified name to the `log4j2.xml` file to enable verbose logging for Source:
+
+[source,xml,linenums]
+----
+<Loggers>
+   ...
+    <!-- Mule logger -->
+    <AsyncLogger name="org.mule.runtime.core.internal.processor.LoggerMessageProcessor" level="INFO"/>
+    <AsyncLogger name="org.mule.runtime.module.extension.internal.runtime.source.ExtensionMessageSource" level="DEBUG"/>
+    <AsyncRoot level="INFO">
+        <AppenderRef ref="file" />
+    </AsyncRoot>
+</Loggers>
+----
+
+=== Enable Verbose Logging in Anypoint Runtime Manager
+
+The following steps describe how to enable verbose logging using Anypoint Runtime Manager:
+
+. Access Anypoint Runtime Manager.
+. Select the desired application.
+. Go to *Settings* on the left pane.
+. Go to the *Logging* tab.
+
++
+image:enable-sdk-logging-runtime-manager.png[sdk-verbose-logging-runtime-manager]
++
+
+. Add the component's fully qualified name and the logging level (DEBUG or TRACE).
+Refer to the previous section on how to specify the component's fully qualified name.
```

Comment:
We don't want to provide a locator like "previous section" because it creates a locational dependence and blocks reuse. Instead, maybe something like "Refer to the table that...."